### PR TITLE
README Review & Suggestions

### PR DESCRIPTION
**Pull request by abdel**
As part of [this](https://www.pivotaltracker.com/story/show/148280641) task, I went through the getting started steps in the updated README and read the wiki page. I fixed a couple of typos that I noticed along the way, and added a few suggestions. I'll be addressing questions and steps in the task description below.

**1. What is missing from the explanation?**
I think it's explained ok with the README and wiki - there's probably still a lot more to explain under the hood, but I think the general idea and overview are clear from the instructions provided.

**2. What is confusing?**
I found the `Run a Demo` and `Run a Generic Experiment` sections were a bit confusing. It wasn't clear what was the purpose of the `update` signal, and wasn't really sure what was expected to happen when I tried to Update in the GUI? I think that may be due to the GUI being confusing in general for me personally which makes it difficult to navigate and figure out what's happening.
 
**3. Do you know what options you have next?**
Kind of - while it explains what entities and data are used for - it's still actually hard to figure out what I actually need to do to setup a custom experiment. Maybe more in-depth explanations of the JSON files and how they are used will be useful for this.

**Next Steps**
- [x] use GUI
- [ ] create your own Demo to implement a trivial algorithm
- [x] run an existing experiment (using experiment-definitions and run-framework)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,11 +91,11 @@ If using PostgreSQL, to administer the database manually (not essential, but use
 We provide project files to help you build and browse code using IntelliJ IDEA.
 If you wish to take advantage of this convenience, you should also install IntelliJ.
 
-
 Then:
 * Pull the repository
-* Set variables. Duplicate `/resources/variables-template.sh`, and overwrite with values suitable for your environment. Copy it to a convenient location and set an environmental variable VARIABLES_FILE to point to it using the full path. We recommend you set that up in .bashrc so that it is always definted correctly.
-* The favoured (and our current) approach is to use 'in memory' persistence, specified in `node.properties` in the working folder. However, postgres is an option. If using postgres, setup and run the db. Run `/bin/db/setup.sh`
+* Build the project using `mvn package -f code/core/pom.xml`
```
Comment (abdel) [inline README.md]: I remember building the project originally, maybe from an outdated README. Is this a necessary step, or is the core re-built on experiment runs anyway?

Comment (drawlinson) [inline README.md]: Hi Abdel, thanks for the feedback. I don't know the answer to the readme / mvn package step. I think it depends on how it is being run and deployed. I will have to have a chat with Gideon about this and we'll also talk about the more confusing bits of the intro. Perhaps I will add some info about the update() call and so on. Will get back to you.

Comment (affogato) [inline README.md]: The project is built with maven, whether you do that on command line or through the IDE.

When you do it on the command line, then you use the command `mvn package` from the project folder.

Not sure what that line is with the -f, i guess you can use that command if you're not IN the project folder already.

Comment (abdel) [inline README.md]: I used `-f` to make it clearer relative to the steps. If someone pulls the repository, and naturally they'll be inside the root path of the repo.

**Pull request merged**